aten/src/ATen/native/ConvolutionMM2d.cpp

```diff
@@ -634,6 +643,7 @@ Tensor slow_conv2d_forward_cpu(
       padding,
       output);
 
+
```

Comment:
There are probably more cases where this is happening, but we should make sure this is the right approach first.